### ISSUE TITLE
Several audits do not provide critical information

### ISSUE THREAD

**Issue by sgomes** [GoogleChrome/lighthouse#2223]
Lighthouse audits are failing to provide some critical information on where best practices are being broken on the audited page.

Here are some issues I've found so far:
- The color contrast audit provides no information on the failing elements
- The "rel='noopener'" audit provides no information on the failing elements
- The "images have alt" audit provides no information on the failing elements
- The "uses passive listeners" audit no longer provides a code snippet or column number, only line number

**Comment by paulirish** [comment]
Thanks for reporting! Really helpful to have this tracked in an issue.

**Comment by brendankenny** [comment]
Obviously it's been quite some time here :) Not sure what was going on with the a11y audits at the time, but they all have info on the failing element now.
